client/state/current-user/test/selectors.js

```diff
@@ -98,6 +99,48 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getCurrentUserIsRTL', () => {
+		it( 'should return null if the current user is not set', () => {
+			const isRTL = getCurrentUserIsRTL( {
+				currentUser: {
+					id: null
+				}
+			} );
+
+			expect( isRTL ).to.be.null;
+		} );
+
+		it( 'should return null if the current user isRTL slug is not set', () => {
+			const isRTL = getCurrentUserIsRTL( {
+				users: {
+					items: {
+						73705554: { ID: 73705554, login: 'testonesite2014' }
+					}
+				},
+				currentUser: {
+					id: 73705554
+				}
+			} );
+
+			expect( isRTL ).to.be.null;
+		} );
+
+		it( 'should return the current user isRTL value', () => {
```

Comment:
I know it might look redundant, but it's generally a good idea to include test cases for both the `true` and `false` situations. The more tests, the better 😉